pkg/entities/audience.go

```diff
@@ -32,3 +32,11 @@ type Condition struct {
 	Value                interface{} `json:"value"`
 	StringRepresentation string
 }
+
+// ToOptimizelyAudience converts to OptimizelyAudience
+func (a Audience) ToOptimizelyAudience() OptimizelyAudience {
+	return OptimizelyAudience{
+		ID:         a.ID,
+		Name:       a.Name,
+		Conditions: "TODO"}
```

Comment:
This conditions should just come from your a variable -> a.Conditions, no other work needed here I dont believe unless go has these stored differently. The audiences in OptimizelyExperiment will be the string of audienceConditions (stringify).

Reply:
The stringify should be done at this point.

Reply:
Tagged you in design doc to avoid confusion

Reply:
Thanks!

Reply:
It's stored as a {ConditionTree} so will require a bit of work here!